README.md

```diff
@@ -1,39 +1,50 @@
-[![Stories in Ready](https://badge.waffle.io/pulibrary/cicognara-rails.png?label=ready&title=Ready)](https://waffle.io/pulibrary/cicognara-rails)
+# Cicognara
 
-# cicognara-rails
-Rails app for managing metadata for the Digital Cicogara Library
+An application for indexing and displaying IIIF Manifests from multiple partners
+in an effort to create a high quality digital version of Count Leopoldo Cicognara's private
+book collection.
 
-# To Run:
-```
-# this runs solr and indexes the fixture data
-rake server
-```
+[![CircleCI](https://circleci.com/gh/pulibrary/cicognara-rails.svg?style=svg)](https://circleci.com/gh/pulibrary/cicognara-rails)
+[![Coverage Status](https://coveralls.io/repos/pulibrary/cicognara-rails/badge.svg?branch=master&service=github)](https://coveralls.io/github/pulibrary/cicognara-rails?branch=master)
 
-# To index the full Catalogo:
-```
-TEIPATH=/path/to/Catalogo_Cicognara.tei.xml MARCPATH=/path/to/cicognara.mrx.xml rake tei:index
-```
 
-# To create html partials of full Catalogo:
-```
-# creates partials at default path app/views/pages/catalogo
-TEIPATH=/path/to/cicognara.tei.xml rake tei:partials
-```
-A non-default partials path can be specified:
-```
- PARTIALSPATH=app/views/pages/NON-DEFAULT-FOLDER rake tei:partials
-```
+## Initial Setup
 
-# To create the initial admin user, create a user account and then make the last user an admin:
-```
-rake set_admin_role
-```
-You can also specify an email address to make a particular user an admin:
-```
-EMAIL=user@example.org rake set_admin_role
+```sh
+git clone https://github.com/pulibrary/cicognara-rails.git
+cd cicognara-rails
+bundle install
```

Comment:
then yarn install here?

Reply:
Project's too old, it doesn't use yarn/npm/webpacker yet.

Reply:
Ah, okay, then the line below that mentions `yarn install` needs to be changed.

Reply:
Yup, fixed. Added info about getty import while I was it.